turbo.json

```diff
@@ -83,27 +83,31 @@
       ]
     },
     "lint:js": {
-      "cache": false,
+      "inputs": [
+        "!styles/**/*.css",
+        "!components/**/*.css",
+        "!layouts/**/*.css",
+        "!*.md",
+        "!pages/**/*.{md,mdx}"
```

Comment:
These glob patterns are the reverse of the `inputs` for the other two tasks below.